tests/modules/test_catalog.py

```diff
@@ -0,0 +1,164 @@
+
```

Comment:
Can we test for task id is returned like for the node tests?

Reply:
Just added those. Didn't realize we are checking those in node
Good catch!